### PR TITLE
Fix building from build.cmd on a path containing spaces

### PR DESCRIPTION
**Pull request by danielrbradley**
Add additional quoting when passing the initial command to powershell.

Currently, running `build.cmd` will fail if the path contains spaces.

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -3,5 +3,5 @@
 REM Copyright (c) .NET Foundation and contributors. All rights reserved.
 REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-powershell -NoProfile -NoLogo -Command "%~dp0build_projects\dotnet-cli-build\build.ps1 %*; exit $LastExitCode;"
+powershell -NoProfile -NoLogo -Command "& \"%~dp0build_projects\dotnet-cli-build\build.ps1\" %*; exit $LastExitCode;"
```
Comment (eerhardt) [inline build.cmd]: Have you tried build.sh?  Does it have the same issue?

Comment (danielrbradley) [inline build.cmd]: Nope - fraid I've not got an environment to test that.

**Comment by eerhardt** [comment]
http://dotnet-ci.cloudapp.net/job/dotnet_cli/job/rel_1.0.0/job/debug_ubuntu16.04_x64_prtest/96/

A bunch of "incremental build" flaky tests.

test Ubuntu16.04 x64 Debug Build

**Comment by eerhardt** [comment]
:shipit:

**Pull request merged**